cider/tests/utils/CiderQueryRunner.cpp

```diff
@@ -230,11 +230,11 @@ std::vector<CiderBatch> CiderQueryRunner::handleRes(
   std::vector<CiderBatch> res;
   auto has_more_output = CiderRuntimeModule::ReturnCode::kMoreOutput;
   while (has_more_output == CiderRuntimeModule::ReturnCode::kMoreOutput) {
-    std::vector<const int8_t*> out_col_buffers(column_num);
-    for (size_t i = 0; i < column_num; ++i) {
-      size_t type_bytes = schema->GetColumnTypeSize(i);
-      out_col_buffers[i] = new int8_t[type_bytes * max_output_row_num];
-    }
+    // std::vector<const int8_t*> out_col_buffers(column_num);
```

Comment:
Q: why do we need to comment this out?

Reply:
These codes are useless so we should remove it directly. Will update this.